pkg/clients/clients.go

```diff
@@ -380,6 +380,8 @@ func GetTestClients(k8sMockObjects []runtime.Object) *Settings {
 		// Velero Client Objects
 		case *velerov1.Backup:
 			veleroClientObjects = append(veleroClientObjects, v)
+		case *srIovV1.SriovNetwork:
```

Comment:
Maybe add a comment so it's easier to identify which objects belong to which clients